libutil/dr_config.c

```diff
@@ -1697,23 +1773,26 @@ dr_register_client(const char *process_name, process_id_t pid, bool global,
     opt_info_alloc = true;
 
     for (i = 0; i < opt_info.num_clients; i++) {
-        if (opt_info.client_opts[i]->id == client_id) {
+        if (opt_info.client_opts[i]->id == client->id &&
```

Comment:
Add a check that ensures that an alternative bitwidth client is not registered prior to the regular client.

Reply:
Looking at it now, it actually doesn't matter: each just turns into a -client_lib* option string to DR.  The only problem is if you do an alt and not a regular, no client will be loaded: but you'll get a fatal error so you'd notice.  Or do you think we should artificially require a regular first?

Reply:
dedicated check might be more user friendly for not so experienced users, but just my 2c

Reply:
So you're voting to require an alt client to have the same ID as a primary, and to require a primary to be registered first.  (All if using drconfiglib...raw DR options have no restrictions.)

Reply:
Done.